### DISCUSSION TITLE
How to show no picture?

### DISCUSSION THREAD

**Discussion by thecarpetjasp** [Windos/BurntToast#198] [Q&A] [answered]
Is there a way to show a notification without a picture? If I just leave out the line for adding a picture - it just adds the burnt toast image by default. Is there a way to do this without any picture?

**Comment by thecarpetjasp** [comment, marked as answer]
I figured it myself.

For anyone wondering - the solution:

`-AppLogo false`

**Reply by Windos** [reply]
Just to add to this, the `-AppLogo` parameter is expecting a path to an image to override the default (which is configured in the config.json file inside the module directory), so putting anything here that doesn't point to a valid image will result in no App Logo Override... but will also output a warning.

Also, you can't do `$null` as that will just use the default app logo.

![image](https://user-images.githubusercontent.com/6955786/159110921-7d418d03-b548-478e-a5a8-5abfea564e03.png)

v1 won't have a default app logo (that's the currect thinking) so this will be less of an issue then.

**Reply by ZeroNotHero** [reply]
false, $false, $null didn't work for me. I had to use: "-AppLogo ." and use just a period for the actual value. Bug?

**Reply by thecarpetjasp** [reply]
That's great to hear :) I think no image being the default is definitely the right way to go

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
